### PR TITLE
Introduce `convert_to_coding_scheme` and make `coding_scheme` deprecated in CoNLL2000DatasetReader

### PR DESCRIPTION
**Pull request by himkt**
Related to https://github.com/allenai/allennlp/pull/4960.

This PR also introduces `convert_to_coding_scheme` to `Conll200DatasetReader` as in `Conll2003DatasetReader` of `allennlp`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- `coding_scheme` parameter is now deprecated in `Conll2000DatasetReader`, please use `convert_to_coding_scheme` instead.
```
Comment (epwalsh) [inline CHANGELOG.md:10]:
```suggestion
### Changed

- `coding_scheme` parameter is now deprecated in `Conll2000DatasetReader`, please use `convert_to_coding_scheme` instead.
```

**Review by epwalsh — CHANGES_REQUESTED** [review]
Thanks @himkt! Looks like you'll need to run `black .` to fix formatting.

File: CHANGELOG.md (hunk 1 above)
Comment (himkt) [reply CHANGELOG.md:10]: Ah, I finally understand that the level of `Unreleased` is not the same as `Changed`.

**Comment by himkt** [comment]
Sorry for forgetting the code-format. I ran `black`.

**Review by epwalsh — APPROVED** [review]
Looks great @himkt! Nice job with the tests 💯

**Pull request merged**